Indian Start-up Ecosystem Funding Analysis (2018-2021)

Business Understanding

Summary of the Task:
This project involves analyzing the funding received by start-ups in India from 2018 to 2021. The goal is to investigate the Indian start-up ecosystem and propose strategic recommendations based on data-driven insights. The datasets are provided for each year, and the analysis will cover start-up details, funding amounts, and investors' information. Data is stored across various sources, and it is crucial to gather, clean, and analyze this data to derive meaningful insights


Project Name:
Indian Start-up Funding Project (2018-2021)

Summary of the Task:
This project involves analyzing the funding received by start-ups in India from 2018 to 2021. The goal is to investigate the Indian start-up ecosystem and propose strategic recommendations based on data-driven insights. The datasets are provided for each year, and the analysis will cover start-up details, funding amounts, and investors' information. Data is stored across various sources, and it is crucial to gather, clean, and analyze this data to derive meaningful insights.

Libraries and Packages:
pandas for data manipulation and analysis
numpy for numerical operations
pyodbc for database connectivity
sqlalchemy for database ORM (optional)
matplotlib and seaborn for data visualization
scikit-learn for machine learning (if applicable)
python-dotenv for managing environment variables
requests for handling HTTP requests (if needed)
os and pathlib for handling file paths and directories

## Business Questions

1.What sectors have shown the highest growth in terms of funding received over the past four years?

2.What geographical regions within India have emerged as the primary hubs for startup activity and investment, and what factors contribute to their prominence?

3.Are there any notable differences in funding patterns between early-stage startups and more established companies?

4.Which sectors recieve the lowest level of funding and which sectors recieve the highest levels of funding in India and what factors contribute to this?

5.Which investors have more impact on startups over the years?

6.What are the key characteristics of startups that successfully secure funding, and how do they differ from those that struggle to attract investment?

1. Sectors with Highest Growth: This question helps identify the sectors that are experiencing rapid growth in terms of funding received, providing valuable insights into where investor interest and capital are flowing. Understanding these sectors can help investors identify potential high-growth opportunities for investment.

2. Geographical Regions for Startup Activity: Understanding the primary hubs for startup activity and investment within India helps investors gauge where the most vibrant ecosystems are located. Factors contributing to their prominence, such as infrastructure, government support, and access to talent, can influence investment decisions and strategies.

3. Funding Patterns Across Startup Stages: Comparing funding patterns between early-stage startups and more established companies helps investors understand how investment behavior varies depending on the maturity and growth stage of the startup. This insight can inform investment strategies tailored to different stages of the startup lifecycle.

4. Sectorial Funding Disparities: Identifying sectors with the lowest and highest levels of funding sheds light on where capital is concentrated and where there may be untapped opportunities. Understanding the factors contributing to these disparities can help investors assess sector-specific risks and opportunities.

5. Impactful Investors: Analyzing the influence of different investors on startups over the years provides insights into which investors have been most active and successful in driving startup growth. This understanding can help investors identify potential partners or co-investors and assess the reputations and track records of different investment firms.

6. Characteristics of Funded Startups: Identifying key characteristics shared by startups that successfully secure funding helps investors understand what factors contribute to investment readiness and attractiveness. Contrasting these characteristics with those of startups that struggle to attract investment can provide valuable lessons for entrepreneurs and investors alike.

Null Hypothesis(Ho): There is no significant difference in the amount of funding between startups in particular "location".

Alternative Hypothesis(Ha): There is a significant difference in the amount of funding between startups in "Blocation".

In [193]:
# Data manipulation and analysis
import pandas as pd
import numpy as np

# Database connectivity
import pyodbc

# Database ORM (optional)
from sqlalchemy import create_engine

# Data visualization
import matplotlib.pyplot as plt
import seaborn as sns

# Machine learning (if applicable)
from sklearn.model_selection import train_test_split
from sklearn.preprocessing import StandardScaler
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import classification_report, confusion_matrix

# Managing environment variables
from dotenv import dotenv_values

# Handling HTTP requests (if needed)
import requests

# Handling file paths and directories
import os
from pathlib import Path


import warnings 

warnings.filterwarnings('ignore')

Loading Data to Python VSO Environment:

1. Database Connection (2020 and 2021 Data):

In [194]:
# Load environment variables from .env file into a dictionary
environment_variables = dotenv_values('.env')

# Get the values for the credentials you set in the '.env' file
server = environment_variables.get("SERVER")
database = environment_variables.get("DATABASE")
username = environment_variables.get("UID")
password = environment_variables.get("PWD")


In [195]:
# Create a connection string
connection_string = f"DRIVER={{SQL Server}};SERVER={server};DATABASE={database};UID={username};PWD={password};MARS_Connection=yes;MinProtocolVersion=TLSv1.2;"

In [196]:
# Use the connect method of the pyodbc library and pass in the connection string.
# This will connect to the server and might take a few seconds to be complete. 
# Check your internet connection if it takes more time than necessary

connection = pyodbc.connect(connection_string)





In [197]:
# Now the sql query to get the data is what what you see below. 


#query = "SELECT * FROM LP2_Telco_churn_first_3000"

# Note that you will not have permissions to insert delete or update this database table. 
# select data from 2020

query = "SELECT * FROM LP1_startup_funding2020"

data20 = pd.read_sql(query, connection)
data20.head()

Company_Brand  Founded HeadQuarter              Sector  \
0    Aqgromalin   2019.0     Chennai            AgriTech   
1      Krayonnz   2019.0   Bangalore              EdTech   
2  PadCare Labs   2018.0        Pune  Hygiene management   
3         NCOME   2020.0   New Delhi              Escrow   
4    Gramophone   2016.0      Indore            AgriTech   

                                        What_it_does  \
0                       Cultivating Ideas for Profit   
1  An academy-guardian-scholar centric ecosystem ...   
2   Converting bio-hazardous waste to harmless waste   
3                       Escrow-as-a-service platform   
4  Gramophone is an AgTech platform enabling acce...   

                                            Founders  \
0                    Prasanna Manogaran, Bharani C L   
1                   Saurabh Dixit, Gurudutt Upadhyay   
2                                    Ajinkya Dhariya   
3                                      Ritesh Tiwari   
4  Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   

                              Investor    Amount     Stage column10  
0                      Angel investors  200000.0      None     None  
1                      GSF Accelerator  100000.0  Pre-seed     None  
2                       Venture Center       NaN  Pre-seed     None  
3  Venture Catalysts, PointOne Capital  400000.0      None     None  
4  Siana Capital Management, Info Edge  340000.0      None     None

In [198]:
data20.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1055 non-null   object 
 1   Founded        842 non-null    float64
 2   HeadQuarter    961 non-null    object 
 3   Sector         1042 non-null   object 
 4   What_it_does   1055 non-null   object 
 5   Founders       1043 non-null   object 
 6   Investor       1017 non-null   object 
 7   Amount         801 non-null    float64
 8   Stage          591 non-null    object 
 9   column10       2 non-null      object 
dtypes: float64(2), object(8)
memory usage: 82.5+ KB


In [199]:
data20.shape

(1055, 10)

In [200]:
# creating a column to identify each dataset by addition of data year

data20['Funding_Year'] = 2020

#Change the funding year to integer type

data20['Funding_Year'] = data20['Funding_Year'].astype(int)

data20.info()

data20.head()


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 11 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1055 non-null   object 
 1   Founded        842 non-null    float64
 2   HeadQuarter    961 non-null    object 
 3   Sector         1042 non-null   object 
 4   What_it_does   1055 non-null   object 
 5   Founders       1043 non-null   object 
 6   Investor       1017 non-null   object 
 7   Amount         801 non-null    float64
 8   Stage          591 non-null    object 
 9   column10       2 non-null      object 
 10  Funding_Year   1055 non-null   int32  
dtypes: float64(2), int32(1), object(8)
memory usage: 86.7+ KB


Company_Brand  Founded HeadQuarter              Sector  \
0    Aqgromalin   2019.0     Chennai            AgriTech   
1      Krayonnz   2019.0   Bangalore              EdTech   
2  PadCare Labs   2018.0        Pune  Hygiene management   
3         NCOME   2020.0   New Delhi              Escrow   
4    Gramophone   2016.0      Indore            AgriTech   

                                        What_it_does  \
0                       Cultivating Ideas for Profit   
1  An academy-guardian-scholar centric ecosystem ...   
2   Converting bio-hazardous waste to harmless waste   
3                       Escrow-as-a-service platform   
4  Gramophone is an AgTech platform enabling acce...   

                                            Founders  \
0                    Prasanna Manogaran, Bharani C L   
1                   Saurabh Dixit, Gurudutt Upadhyay   
2                                    Ajinkya Dhariya   
3                                      Ritesh Tiwari   
4  Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   

                              Investor    Amount     Stage column10  \
0                      Angel investors  200000.0      None     None   
1                      GSF Accelerator  100000.0  Pre-seed     None   
2                       Venture Center       NaN  Pre-seed     None   
3  Venture Catalysts, PointOne Capital  400000.0      None     None   
4  Siana Capital Management, Info Edge  340000.0      None     None   

   Funding_Year  
0          2020  
1          2020  
2          2020  
3          2020  
4          2020

In [201]:
data20.shape

(1055, 11)

In [202]:
#printing columns to compare if the column names are matching
print(data20.columns)

Index(['Company_Brand', 'Founded', 'HeadQuarter', 'Sector', 'What_it_does',
       'Founders', 'Investor', 'Amount', 'Stage', 'column10', 'Funding_Year'],
      dtype='object')


In [203]:
# Renaming some columns

data20.rename(columns = {'Company_Brand' :'Company_Name'}, inplace =True)

data20.rename(columns = {'HeadQuarter': 'Location'}, inplace =True)

data20.head()

Company_Name  Founded   Location              Sector  \
0    Aqgromalin   2019.0    Chennai            AgriTech   
1      Krayonnz   2019.0  Bangalore              EdTech   
2  PadCare Labs   2018.0       Pune  Hygiene management   
3         NCOME   2020.0  New Delhi              Escrow   
4    Gramophone   2016.0     Indore            AgriTech   

                                        What_it_does  \
0                       Cultivating Ideas for Profit   
1  An academy-guardian-scholar centric ecosystem ...   
2   Converting bio-hazardous waste to harmless waste   
3                       Escrow-as-a-service platform   
4  Gramophone is an AgTech platform enabling acce...   

                                            Founders  \
0                    Prasanna Manogaran, Bharani C L   
1                   Saurabh Dixit, Gurudutt Upadhyay   
2                                    Ajinkya Dhariya   
3                                      Ritesh Tiwari   
4  Ashish Rajan Singh, Harshit Gupta, Nishant Mah...   

                              Investor    Amount     Stage column10  \
0                      Angel investors  200000.0      None     None   
1                      GSF Accelerator  100000.0  Pre-seed     None   
2                       Venture Center       NaN  Pre-seed     None   
3  Venture Catalysts, PointOne Capital  400000.0      None     None   
4  Siana Capital Management, Info Edge  340000.0      None     None   

   Funding_Year  
0          2020  
1          2020  
2          2020  
3          2020  
4          2020

In [204]:
#select specific columns
data20 = data20[['Company_Name', 'Founded','Location','Sector','Investor','Amount','Stage','Funding_Year']]
                
data20.head() 

Company_Name  Founded   Location              Sector  \
0    Aqgromalin   2019.0    Chennai            AgriTech   
1      Krayonnz   2019.0  Bangalore              EdTech   
2  PadCare Labs   2018.0       Pune  Hygiene management   
3         NCOME   2020.0  New Delhi              Escrow   
4    Gramophone   2016.0     Indore            AgriTech   

                              Investor    Amount     Stage  Funding_Year  
0                      Angel investors  200000.0      None          2020  
1                      GSF Accelerator  100000.0  Pre-seed          2020  
2                       Venture Center       NaN  Pre-seed          2020  
3  Venture Catalysts, PointOne Capital  400000.0      None          2020  
4  Siana Capital Management, Info Edge  340000.0      None          2020

In [205]:
# Converting the funded column to numeric data
data20['Founded'] = pd.to_numeric(data20['Founded'], errors='coerce').convert_dtypes(int)

In [206]:
# Converting the Amount column to a numeric, there the need to remove some symbols including commas and currency

data20['Amount'] = data20['Amount'].apply(lambda x:str(x).replace('$', ''))

data20['Amount'] = data20['Amount'].apply(lambda x:str(x).replace(',', ''))

data20['Amount'] = data20['Amount'].replace('—', np.nan)




In [207]:
#Find the number of rows with undisclosed amounts 

index1 = data20.index[data20['Amount']=='Undisclosed']

print('The total number of undisclosed records is', len(index1))

The total number of undisclosed records is 0


In [208]:
# convert undisclosed to NAN
data20['Amount'] = data20['Amount'].replace('Undisclosed', np.nan)

In [209]:
#print a summary information on the 2020 data 
data20.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 8 columns):
 #   Column        Non-Null Count  Dtype 
---  ------        --------------  ----- 
 0   Company_Name  1055 non-null   object
 1   Founded       842 non-null    Int64 
 2   Location      961 non-null    object
 3   Sector        1042 non-null   object
 4   Investor      1017 non-null   object
 5   Amount        1055 non-null   object
 6   Stage         591 non-null    object
 7   Funding_Year  1055 non-null   int32 
dtypes: Int64(1), int32(1), object(6)
memory usage: 63.0+ KB


In [210]:
#Find the row with 887000 23000000 in the amount section
index1 = data20.index[data20['Amount']=='887000 23000000']
index1

Index([], dtype='int64')

In [211]:
#replace the values with the average 
avg = str((887000+23000000)/2)
data20.at[465, 'Amount'] = avg 


In [212]:
#print the row record to confirm
print(data20.iloc[(465)])

Company_Name    True Balance
Founded                 2014
Location            Gurugram
Sector               Finance
Investor         Balancehero
Amount            11943500.0
Stage               Series C
Funding_Year            2020
Name: 465, dtype: object


In [213]:

#Find the row with 800000000 to 850000000 in the amount section
index2 = data20.index[data20['Amount']=='800000000 to 850000000']

In [214]:
#replace the values with the average 
avg = str((800000000+850000000)/2)

data20.at[472, 'Amount'] = avg 

In [215]:
#print the row record to confirm 
print(data20.iloc[(472)])

Company_Name                                             Eruditus
Founded                                                      2010
Location                                                   Mumbai
Sector                                                  Education
Investor        Bertelsmann India Investments, Sequoia Capital...
Amount                                                825000000.0
Stage                                                        None
Funding_Year                                                 2020
Name: 472, dtype: object


In [216]:
#Convert the Amount column to numeric 

data20['Amount'] = pd.to_numeric(data20['Amount'], errors='coerce')

In [217]:
#print a summary information on the 2020 data 
data20.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1055 entries, 0 to 1054
Data columns (total 8 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   Company_Name  1055 non-null   object 
 1   Founded       842 non-null    Int64  
 2   Location      961 non-null    object 
 3   Sector        1042 non-null   object 
 4   Investor      1017 non-null   object 
 5   Amount        803 non-null    float64
 6   Stage         591 non-null    object 
 7   Funding_Year  1055 non-null   int32  
dtypes: Int64(1), float64(1), int32(1), object(5)
memory usage: 63.0+ KB


In [218]:
duplicates = data20[data20.duplicated()]

duplicates

Company_Name  Founded   Location                 Sector  \
145    Krimanshi     2015    Jodhpur  Biotechnology company   
205        Nykaa     2012     Mumbai              Cosmetics   
362       Byju’s     2011  Bangalore                 EdTech   

                                           Investor       Amount Stage  \
145  Rajasthan Venture Capital Fund, AIM Smart City     600000.0  Seed   
205                        Alia Bhatt, Katrina Kaif          NaN  None   
362           Owl Ventures, Tiger Global Management  500000000.0  None   

     Funding_Year  
145          2020  
205          2020  
362          2020

In [221]:
#drop all duplicates and leave only one record 

data20 = data20.drop_duplicates(keep='first')

In [222]:
#Check the 2020 datatset information to confirm the datatypes 
data20.info()

<class 'pandas.core.frame.DataFrame'>
Index: 1052 entries, 0 to 1054
Data columns (total 8 columns):
 #   Column        Non-Null Count  Dtype  
---  ------        --------------  -----  
 0   Company_Name  1052 non-null   object 
 1   Founded       839 non-null    Int64  
 2   Location      958 non-null    object 
 3   Sector        1039 non-null   object 
 4   Investor      1014 non-null   object 
 5   Amount        801 non-null    float64
 6   Stage         590 non-null    object 
 7   Funding_Year  1052 non-null   int32  
dtypes: Int64(1), float64(1), int32(1), object(5)
memory usage: 70.9+ KB


In [224]:
#Check the first set of row 
data20.head()

Company_Name  Founded   Location              Sector  \
0    Aqgromalin     2019    Chennai            AgriTech   
1      Krayonnz     2019  Bangalore              EdTech   
2  PadCare Labs     2018       Pune  Hygiene management   
3         NCOME     2020  New Delhi              Escrow   
4    Gramophone     2016     Indore            AgriTech   

                              Investor    Amount     Stage  Funding_Year  
0                      Angel investors  200000.0      None          2020  
1                      GSF Accelerator  100000.0  Pre-seed          2020  
2                       Venture Center       NaN  Pre-seed          2020  
3  Venture Catalysts, PointOne Capital  400000.0      None          2020  
4  Siana Capital Management, Info Edge  340000.0      None          2020

In [ ]:
# select data from 2021

query = "SELECT * FROM LP1_startup_funding2021"

data21 = pd.read_sql(query, connection)
data21.head()

Company_Brand  Founded HeadQuarter          Sector  \
0  Unbox Robotics   2019.0   Bangalore      AI startup   
1          upGrad   2015.0      Mumbai          EdTech   
2     Lead School   2012.0      Mumbai          EdTech   
3         Bizongo   2015.0      Mumbai  B2B E-commerce   
4        FypMoney   2021.0    Gurugram         FinTech   

                                        What_it_does  \
0  Unbox Robotics builds on-demand AI-driven ware...   
1     UpGrad is an online higher education platform.   
2  LEAD School offers technology based school tra...   
3  Bizongo is a business-to-business online marke...   
4  FypMoney is Digital NEO Bank for Teenagers, em...   

                                            Founders  \
0                        Pramod Ghadge, Shahid Memon   
1  Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                         Smita Deorah, Sumeet Mehta   
3            Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                      Kapil Banwari   

                                        Investor        Amount         Stage  
0                    BEENEXT, Entrepreneur First    $1,200,000  Pre-series A  
1       Unilazer Ventures, IIFL Asset Management  $120,000,000          None  
2               GSV Ventures, Westbridge Capital   $30,000,000      Series D  
3                         CDC Group, IDG Capital   $51,000,000      Series C  
4  Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000          Seed

In [ ]:
data21.shape

(1209, 9)

In [ ]:
data21.info()


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1209 entries, 0 to 1208
Data columns (total 9 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1209 non-null   object 
 1   Founded        1208 non-null   float64
 2   HeadQuarter    1208 non-null   object 
 3   Sector         1209 non-null   object 
 4   What_it_does   1209 non-null   object 
 5   Founders       1205 non-null   object 
 6   Investor       1147 non-null   object 
 7   Amount         1206 non-null   object 
 8   Stage          781 non-null    object 
dtypes: float64(1), object(8)
memory usage: 85.1+ KB


In [ ]:
# creating a column to identify each dataset by addition of data year

data21['Funding_Year'] = 2021

# change the Funding_Year to interger type

data21['Funding_Year'] = data21['Funding_Year'].astype(int)

data21.info()

data21.head()






<class 'pandas.core.frame.DataFrame'>
RangeIndex: 1209 entries, 0 to 1208
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company_Brand  1209 non-null   object 
 1   Founded        1208 non-null   float64
 2   HeadQuarter    1208 non-null   object 
 3   Sector         1209 non-null   object 
 4   What_it_does   1209 non-null   object 
 5   Founders       1205 non-null   object 
 6   Investor       1147 non-null   object 
 7   Amount         1206 non-null   object 
 8   Stage          781 non-null    object 
 9   Funding_Year   1209 non-null   int32  
dtypes: float64(1), int32(1), object(8)
memory usage: 89.9+ KB


Company_Brand  Founded HeadQuarter          Sector  \
0  Unbox Robotics   2019.0   Bangalore      AI startup   
1          upGrad   2015.0      Mumbai          EdTech   
2     Lead School   2012.0      Mumbai          EdTech   
3         Bizongo   2015.0      Mumbai  B2B E-commerce   
4        FypMoney   2021.0    Gurugram         FinTech   

                                        What_it_does  \
0  Unbox Robotics builds on-demand AI-driven ware...   
1     UpGrad is an online higher education platform.   
2  LEAD School offers technology based school tra...   
3  Bizongo is a business-to-business online marke...   
4  FypMoney is Digital NEO Bank for Teenagers, em...   

                                            Founders  \
0                        Pramod Ghadge, Shahid Memon   
1  Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                         Smita Deorah, Sumeet Mehta   
3            Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                      Kapil Banwari   

                                        Investor        Amount         Stage  \
0                    BEENEXT, Entrepreneur First    $1,200,000  Pre-series A   
1       Unilazer Ventures, IIFL Asset Management  $120,000,000          None   
2               GSV Ventures, Westbridge Capital   $30,000,000      Series D   
3                         CDC Group, IDG Capital   $51,000,000      Series C   
4  Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000          Seed   

   Funding_Year  
0          2021  
1          2021  
2          2021  
3          2021  
4          2021

In [ ]:
#printing columns to compare if the column names are matching

print(data21.columns)






Index(['Company_Name', 'Founded', 'Location', 'Sector', 'What_it_does',
       'Founders', 'Investor', 'Amount', 'Stage', 'Funding_Year'],
      dtype='object')


In [ ]:
# Renaming some columns

data21.rename(columns = {'Company_Brand' :'Company_Name'}, inplace =True)

data21.rename(columns = {'HeadQuarter': 'Location'}, inplace =True)

data21.head()

Company_Name  Founded   Location          Sector  \
0  Unbox Robotics   2019.0  Bangalore      AI startup   
1          upGrad   2015.0     Mumbai          EdTech   
2     Lead School   2012.0     Mumbai          EdTech   
3         Bizongo   2015.0     Mumbai  B2B E-commerce   
4        FypMoney   2021.0   Gurugram         FinTech   

                                        What_it_does  \
0  Unbox Robotics builds on-demand AI-driven ware...   
1     UpGrad is an online higher education platform.   
2  LEAD School offers technology based school tra...   
3  Bizongo is a business-to-business online marke...   
4  FypMoney is Digital NEO Bank for Teenagers, em...   

                                            Founders  \
0                        Pramod Ghadge, Shahid Memon   
1  Mayank Kumar, Phalgun Kompalli, Ravijot Chugh,...   
2                         Smita Deorah, Sumeet Mehta   
3            Aniket Deb, Ankit Tomar, Sachin Agrawal   
4                                      Kapil Banwari   

                                        Investor        Amount         Stage  \
0                    BEENEXT, Entrepreneur First    $1,200,000  Pre-series A   
1       Unilazer Ventures, IIFL Asset Management  $120,000,000          None   
2               GSV Ventures, Westbridge Capital   $30,000,000      Series D   
3                         CDC Group, IDG Capital   $51,000,000      Series C   
4  Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000          Seed   

   Funding_Year  
0          2021  
1          2021  
2          2021  
3          2021  
4          2021

In [ ]:
#select specific columns
data21 = data21[['Company_Name', 'Founded','Location','Sector','Investor','Amount','Stage','Funding_Year']]
                
data21.head() 

Company_Name  Founded   Location          Sector  \
0  Unbox Robotics   2019.0  Bangalore      AI startup   
1          upGrad   2015.0     Mumbai          EdTech   
2     Lead School   2012.0     Mumbai          EdTech   
3         Bizongo   2015.0     Mumbai  B2B E-commerce   
4        FypMoney   2021.0   Gurugram         FinTech   

                                        Investor        Amount         Stage  \
0                    BEENEXT, Entrepreneur First    $1,200,000  Pre-series A   
1       Unilazer Ventures, IIFL Asset Management  $120,000,000          None   
2               GSV Ventures, Westbridge Capital   $30,000,000      Series D   
3                         CDC Group, IDG Capital   $51,000,000      Series C   
4  Liberatha Kallat, Mukesh Yadav, Dinesh Nagpal    $2,000,000          Seed   

   Funding_Year  
0          2021  
1          2021  
2          2021  
3          2021  
4          2021

Loading Data to Python VSO Environment:

2. Database Connection (2019 Data):

In [ ]:
# Accessing the data for 2019 can be found in OneDrive. The file name startup_funding2019.csv

data19 = pd.read_csv('startup_funding2019.csv')
data19.head()



Company/Brand  Founded HeadQuarter           Sector  \
0  Bombay Shaving      NaN         NaN        Ecommerce   
1       Ruangguru   2014.0      Mumbai           Edtech   
2        Eduisfun      NaN      Mumbai           Edtech   
3        HomeLane   2014.0     Chennai  Interior design   
4        Nu Genes   2004.0   Telangana         AgriTech   

                                        What it does  \
0         Provides a range of male grooming products   
1  A learning platform that provides topic-based ...   
2            It aims to make learning fun via games.   
3              Provides interior designing solutions   
4  It is a seed company engaged in production, pr...   

                                Founders  \
0                     Shantanu Deshpande   
1  Adamas Belva Syah Devara, Iman Usman.   
2                          Jatin Solanki   
3           Srikanth Iyer, Rama Harinath   
4                 Narayana Reddy Punyala   

                                            Investor     Amount($)  \
0                               Sixth Sense Ventures    $6,300,000   
1                                   General Atlantic  $150,000,000   
2     Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3  Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4           Innovation in Food and Agriculture (IFA)    $6,000,000   

           Stage  
0            NaN  
1       Series C  
2  Fresh funding  
3       Series D  
4            NaN

In [ ]:
data19.shape


(89, 9)

In [ ]:
data19.info()


<class 'pandas.core.frame.DataFrame'>
RangeIndex: 89 entries, 0 to 88
Data columns (total 9 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company/Brand  89 non-null     object 
 1   Founded        60 non-null     float64
 2   HeadQuarter    70 non-null     object 
 3   Sector         84 non-null     object 
 4   What it does   89 non-null     object 
 5   Founders       86 non-null     object 
 6   Investor       89 non-null     object 
 7   Amount($)      89 non-null     object 
 8   Stage          43 non-null     object 
dtypes: float64(1), object(8)
memory usage: 6.4+ KB


In [ ]:
# Creating a column to identify each dataset by addition of data year
data19['Year'] = 2019

data19.head()

Company/Brand  Founded HeadQuarter           Sector  \
0  Bombay Shaving      NaN         NaN        Ecommerce   
1       Ruangguru   2014.0      Mumbai           Edtech   
2        Eduisfun      NaN      Mumbai           Edtech   
3        HomeLane   2014.0     Chennai  Interior design   
4        Nu Genes   2004.0   Telangana         AgriTech   

                                        What it does  \
0         Provides a range of male grooming products   
1  A learning platform that provides topic-based ...   
2            It aims to make learning fun via games.   
3              Provides interior designing solutions   
4  It is a seed company engaged in production, pr...   

                                Founders  \
0                     Shantanu Deshpande   
1  Adamas Belva Syah Devara, Iman Usman.   
2                          Jatin Solanki   
3           Srikanth Iyer, Rama Harinath   
4                 Narayana Reddy Punyala   

                                            Investor     Amount($)  \
0                               Sixth Sense Ventures    $6,300,000   
1                                   General Atlantic  $150,000,000   
2     Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3  Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4           Innovation in Food and Agriculture (IFA)    $6,000,000   

           Stage  Year  
0            NaN  2019  
1       Series C  2019  
2  Fresh funding  2019  
3       Series D  2019  
4            NaN  2019

In [ ]:
#Renaming amount column in dataset 2#

data19 = data19.rename(columns={'Amount($)':'Amount'})
data19

Company/Brand  Founded HeadQuarter                   Sector  \
0    Bombay Shaving      NaN         NaN                Ecommerce   
1         Ruangguru   2014.0      Mumbai                   Edtech   
2          Eduisfun      NaN      Mumbai                   Edtech   
3          HomeLane   2014.0     Chennai          Interior design   
4          Nu Genes   2004.0   Telangana                 AgriTech   
..              ...      ...         ...                      ...   
84     Infra.Market      NaN      Mumbai                Infratech   
85              Oyo   2013.0    Gurugram              Hospitality   
86       GoMechanic   2016.0       Delhi  Automobile & Technology   
87           Spinny   2015.0       Delhi               Automobile   
88  Ess Kay Fincorp      NaN   Rajasthan                  Banking   

                                         What it does  \
0          Provides a range of male grooming products   
1   A learning platform that provides topic-based ...   
2             It aims to make learning fun via games.   
3               Provides interior designing solutions   
4   It is a seed company engaged in production, pr...   
..                                                ...   
84  It connects client requirements to their suppl...   
85                Provides rooms for comfortable stay   
86  Find automobile repair and maintenance service...   
87                                Online car retailer   
88              Organised Non-Banking Finance Company   

                                             Founders  \
0                                  Shantanu Deshpande   
1               Adamas Belva Syah Devara, Iman Usman.   
2                                       Jatin Solanki   
3                        Srikanth Iyer, Rama Harinath   
4                              Narayana Reddy Punyala   
..                                                ...   
84                    Aaditya Sharda, Souvik Sengupta   
85                                     Ritesh Agarwal   
86  Amit Bhasin, Kushal Karwa, Nitin Rana, Rishabh...   
87  Niraj Singh, Ramanshu Mahaur, Ganesh Pawar, Mo...   
88                                     Rajendra Setia   

                                             Investor        Amount  \
0                                Sixth Sense Ventures    $6,300,000   
1                                    General Atlantic  $150,000,000   
2      Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3   Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4            Innovation in Food and Agriculture (IFA)    $6,000,000   
..                                                ...           ...   
84  Tiger Global, Nexus Venture Partners, Accel Pa...   $20,000,000   
85  MyPreferred Transformation, Avendus Finance, S...  $693,000,000   
86                                    Sequoia Capital    $5,000,000   
87  Norwest Venture Partners, General Catalyst, Fu...   $50,000,000   
88     TPG, Norwest Venture Partners, Evolvence India   $33,000,000   

            Stage  Year  
0             NaN  2019  
1        Series C  2019  
2   Fresh funding  2019  
3        Series D  2019  
4             NaN  2019  
..            ...   ...  
84       Series A  2019  
85            NaN  2019  
86       Series B  2019  
87            NaN  2019  
88            NaN  2019  

[89 rows x 10 columns]

In [ ]:
# Define a function to convert rupees to dollars
def convert_to_dollars(Amount):
    if Amount.startswith('₹'):
        return float(Amount[1:]) * 0.0118
    else:
        return Amount
 
# Apply the conversion function to the 'amount' column
data19['Amount'] = data19['Amount'].apply(convert_to_dollars)
data19

Company/Brand  Founded HeadQuarter                   Sector  \
0    Bombay Shaving      NaN         NaN                Ecommerce   
1         Ruangguru   2014.0      Mumbai                   Edtech   
2          Eduisfun      NaN      Mumbai                   Edtech   
3          HomeLane   2014.0     Chennai          Interior design   
4          Nu Genes   2004.0   Telangana                 AgriTech   
..              ...      ...         ...                      ...   
84     Infra.Market      NaN      Mumbai                Infratech   
85              Oyo   2013.0    Gurugram              Hospitality   
86       GoMechanic   2016.0       Delhi  Automobile & Technology   
87           Spinny   2015.0       Delhi               Automobile   
88  Ess Kay Fincorp      NaN   Rajasthan                  Banking   

                                         What it does  \
0          Provides a range of male grooming products   
1   A learning platform that provides topic-based ...   
2             It aims to make learning fun via games.   
3               Provides interior designing solutions   
4   It is a seed company engaged in production, pr...   
..                                                ...   
84  It connects client requirements to their suppl...   
85                Provides rooms for comfortable stay   
86  Find automobile repair and maintenance service...   
87                                Online car retailer   
88              Organised Non-Banking Finance Company   

                                             Founders  \
0                                  Shantanu Deshpande   
1               Adamas Belva Syah Devara, Iman Usman.   
2                                       Jatin Solanki   
3                        Srikanth Iyer, Rama Harinath   
4                              Narayana Reddy Punyala   
..                                                ...   
84                    Aaditya Sharda, Souvik Sengupta   
85                                     Ritesh Agarwal   
86  Amit Bhasin, Kushal Karwa, Nitin Rana, Rishabh...   
87  Niraj Singh, Ramanshu Mahaur, Ganesh Pawar, Mo...   
88                                     Rajendra Setia   

                                             Investor        Amount  \
0                                Sixth Sense Ventures    $6,300,000   
1                                    General Atlantic  $150,000,000   
2      Deepak Parekh, Amitabh Bachchan, Piyush Pandey   $28,000,000   
3   Evolvence India Fund (EIF), Pidilite Group, FJ...   $30,000,000   
4            Innovation in Food and Agriculture (IFA)    $6,000,000   
..                                                ...           ...   
84  Tiger Global, Nexus Venture Partners, Accel Pa...   $20,000,000   
85  MyPreferred Transformation, Avendus Finance, S...  $693,000,000   
86                                    Sequoia Capital    $5,000,000   
87  Norwest Venture Partners, General Catalyst, Fu...   $50,000,000   
88     TPG, Norwest Venture Partners, Evolvence India   $33,000,000   

            Stage  Year  
0             NaN  2019  
1        Series C  2019  
2   Fresh funding  2019  
3        Series D  2019  
4             NaN  2019  
..            ...   ...  
84       Series A  2019  
85            NaN  2019  
86       Series B  2019  
87            NaN  2019  
88            NaN  2019  

[89 rows x 10 columns]

In [ ]:
#removing dollar sign
data19['Amount']= data19['Amount'].str.replace('$', '')
data19

Company/Brand  Founded HeadQuarter                   Sector  \
0    Bombay Shaving      NaN         NaN                Ecommerce   
1         Ruangguru   2014.0      Mumbai                   Edtech   
2          Eduisfun      NaN      Mumbai                   Edtech   
3          HomeLane   2014.0     Chennai          Interior design   
4          Nu Genes   2004.0   Telangana                 AgriTech   
..              ...      ...         ...                      ...   
84     Infra.Market      NaN      Mumbai                Infratech   
85              Oyo   2013.0    Gurugram              Hospitality   
86       GoMechanic   2016.0       Delhi  Automobile & Technology   
87           Spinny   2015.0       Delhi               Automobile   
88  Ess Kay Fincorp      NaN   Rajasthan                  Banking   

                                         What it does  \
0          Provides a range of male grooming products   
1   A learning platform that provides topic-based ...   
2             It aims to make learning fun via games.   
3               Provides interior designing solutions   
4   It is a seed company engaged in production, pr...   
..                                                ...   
84  It connects client requirements to their suppl...   
85                Provides rooms for comfortable stay   
86  Find automobile repair and maintenance service...   
87                                Online car retailer   
88              Organised Non-Banking Finance Company   

                                             Founders  \
0                                  Shantanu Deshpande   
1               Adamas Belva Syah Devara, Iman Usman.   
2                                       Jatin Solanki   
3                        Srikanth Iyer, Rama Harinath   
4                              Narayana Reddy Punyala   
..                                                ...   
84                    Aaditya Sharda, Souvik Sengupta   
85                                     Ritesh Agarwal   
86  Amit Bhasin, Kushal Karwa, Nitin Rana, Rishabh...   
87  Niraj Singh, Ramanshu Mahaur, Ganesh Pawar, Mo...   
88                                     Rajendra Setia   

                                             Investor       Amount  \
0                                Sixth Sense Ventures    6,300,000   
1                                    General Atlantic  150,000,000   
2      Deepak Parekh, Amitabh Bachchan, Piyush Pandey   28,000,000   
3   Evolvence India Fund (EIF), Pidilite Group, FJ...   30,000,000   
4            Innovation in Food and Agriculture (IFA)    6,000,000   
..                                                ...          ...   
84  Tiger Global, Nexus Venture Partners, Accel Pa...   20,000,000   
85  MyPreferred Transformation, Avendus Finance, S...  693,000,000   
86                                    Sequoia Capital    5,000,000   
87  Norwest Venture Partners, General Catalyst, Fu...   50,000,000   
88     TPG, Norwest Venture Partners, Evolvence India   33,000,000   

            Stage  Year  
0             NaN  2019  
1        Series C  2019  
2   Fresh funding  2019  
3        Series D  2019  
4             NaN  2019  
..            ...   ...  
84       Series A  2019  
85            NaN  2019  
86       Series B  2019  
87            NaN  2019  
88            NaN  2019  

[89 rows x 10 columns]

In [ ]:
# remove commas
data19['Amount']= data19['Amount'].str.replace(',', '', regex= True)

data19.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 89 entries, 0 to 88
Data columns (total 10 columns):
 #   Column         Non-Null Count  Dtype  
---  ------         --------------  -----  
 0   Company/Brand  89 non-null     object 
 1   Founded        60 non-null     float64
 2   HeadQuarter    70 non-null     object 
 3   Sector         84 non-null     object 
 4   What it does   89 non-null     object 
 5   Founders       86 non-null     object 
 6   Investor       89 non-null     object 
 7   Amount         89 non-null     object 
 8   Stage          43 non-null     object 
 9   Year           89 non-null     int64  
dtypes: float64(1), int64(1), object(8)
memory usage: 7.1+ KB


In [ ]:
data19['Amount'].unique()

array(['6300000', '150000000', '28000000', '30000000', '6000000',
       'Undisclosed', '1000000', '20000000', '275000000', '22000000',
       '5000000', '140500', '540000000', '15000000', '182700', '12000000',
       '11000000', '15500000', '1500000', '5500000', '2500000', '140000',
       '230000000', '49400000', '32000000', '26000000', '150000',
       '400000', '2000000', '100000000', '8000000', '100000', '50000000',
       '120000000', '4000000', '6800000', '36000000', '5700000',
       '25000000', '600000', '70000000', '60000000', '220000', '2800000',
       '2100000', '7000000', '311000000', '4800000', '693000000',
       '33000000'], dtype=object)

Loading Data to Python VSO Environment:

2. Database Connection (2018 Data):

In [ ]:
# The third data (data for 2018) is hosted on this GitHub Repository, in file called startup_funding2018.csv

data18 = pd.read_csv('startup_funding2018.csv')
data18.head()



Company Name                                           Industry  \
0  TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1  Happy Cow Dairy                               Agriculture, Farming   
2       MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3      PayMe India                        Financial Services, FinTech   
4         Eunimart                 E-Commerce Platforms, Retail, SaaS   

  Round/Series       Amount                          Location  \
0         Seed       250000       Bangalore, Karnataka, India   
1         Seed  ₹40,000,000        Mumbai, Maharashtra, India   
2     Series A  ₹65,000,000           Gurgaon, Haryana, India   
3        Angel      2000000       Noida, Uttar Pradesh, India   
4         Seed            —  Hyderabad, Andhra Pradesh, India   

                                       About Company  
0  TheCollegeFever is a hub for fun, fiesta and f...  
1  A startup which aggregates milk from dairy far...  
2          Leading Online Loans Marketplace in India  
3  PayMe India is an innovative FinTech organizat...  
4  Eunimart is a one stop solution for merchants ...

In [ ]:
data18.shape

(526, 6)

In [ ]:
# Creating a column to identify each dataset by addition of data year

data18['Year'] = 2018

data18.head()

Company Name                                           Industry  \
0  TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1  Happy Cow Dairy                               Agriculture, Farming   
2       MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3      PayMe India                        Financial Services, FinTech   
4         Eunimart                 E-Commerce Platforms, Retail, SaaS   

  Round/Series       Amount                          Location  \
0         Seed       250000       Bangalore, Karnataka, India   
1         Seed  ₹40,000,000        Mumbai, Maharashtra, India   
2     Series A  ₹65,000,000           Gurgaon, Haryana, India   
3        Angel      2000000       Noida, Uttar Pradesh, India   
4         Seed            —  Hyderabad, Andhra Pradesh, India   

                                       About Company  Year  
0  TheCollegeFever is a hub for fun, fiesta and f...  2018  
1  A startup which aggregates milk from dairy far...  2018  
2          Leading Online Loans Marketplace in India  2018  
3  PayMe India is an innovative FinTech organizat...  2018  
4  Eunimart is a one stop solution for merchants ...  2018

In [ ]:
data18.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 526 entries, 0 to 525
Data columns (total 7 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Company Name   526 non-null    object
 1   Industry       526 non-null    object
 2   Round/Series   526 non-null    object
 3   Amount         526 non-null    object
 4   Location       526 non-null    object
 5   About Company  526 non-null    object
 6   Year           526 non-null    int64 
dtypes: int64(1), object(6)
memory usage: 28.9+ KB


In [ ]:
data18['Amount'].unique()

array(['250000', '₹40,000,000', '₹65,000,000', '2000000', '—', '1600000',
       '₹16,000,000', '₹50,000,000', '₹100,000,000', '150000', '1100000',
       '₹500,000', '6000000', '650000', '₹35,000,000', '₹64,000,000',
       '₹20,000,000', '1000000', '5000000', '4000000', '₹30,000,000',
       '2800000', '1700000', '1300000', '₹5,000,000', '₹12,500,000',
       '₹15,000,000', '500000', '₹104,000,000', '₹45,000,000', '13400000',
       '₹25,000,000', '₹26,400,000', '₹8,000,000', '₹60,000', '9000000',
       '100000', '20000', '120000', '₹34,000,000', '₹342,000,000',
       '$143,145', '₹600,000,000', '$742,000,000', '₹1,000,000,000',
       '₹2,000,000,000', '$3,980,000', '$10,000', '₹100,000',
       '₹250,000,000', '$1,000,000,000', '$7,000,000', '$35,000,000',
       '₹550,000,000', '$28,500,000', '$2,000,000', '₹240,000,000',
       '₹120,000,000', '$2,400,000', '$30,000,000', '₹2,500,000,000',
       '$23,000,000', '$150,000', '$11,000,000', '₹44,000,000',
       '$3,240,000', '₹60

In [ ]:
data18[data18['Amount'].str.startswith('$')]

Company Name  \
86                                              WHR   
90                                         SBI Life   
93                 NoPaperForms Solutions Pvt. Ltd.   
95                                       AuthMetrik   
101                                          Swiggy   
102                                      Milkbasket   
104                                           Toppr   
106                                 Vivriti Capital   
108                                     Impact Guru   
114                                       OneAssist   
115         Satin Creditcare Network Limited (SCNL)   
117                                          Moglix   
119                                     MadGuy Labs   
122                                         Shop101   
124                                        Housejoy   
130                                          BYJU’S   
134                                     PerkFinance   
135                                          Roposo   
138                                       Wow Venue   
143                                     MyNewCar.in   
144                           HalaPlay Technologies   
146                                       WheelsEMI   
148                                             OYO   
152                                      Shopkirana   
159                                   Guiddoo World   
160                                      ION Energy   
161                                          UnoTag   
162                                       CivilCops   
166                                       artbandhu   
349  Wicked Ride Adventure Services Private Limited   
351                    ERUDITUS Executive Education   
353                                      Square Off   
354                                       UrbanClap   
355                                         Hospals   
358                                The Souled Store   
360                                         Drivezy   
361                                    Freyr Energy   
362                         NIRAMAI Health Analytix   
365                                         Verloop   
367                                       Paymatrix   
369                                          InCred   
370                                         Vedantu   
372                                Stelis Biopharma   
375                                      Mech Mocha   
377                                         Aldopay   
378                             Detect Technologies   
380                                        CashRich   
384                                         Yaantra   
385                                       TECHBOOZE   
386                                     HealthifyMe   
389    Royal Sundaram General Insurance Co. Limited   
392                                          Spinny   
396                     Mobile Premier League (MPL)   
397                                   WalMart India   
399                                          Meesho   
400                                     BhaskarGuru   
401                                            Gugu   
403                                         Vizzbee   
404                   BIGPHI TECHNOLOGIES PVT. LTD.   

                                              Industry  \
86                 Health Care, Information Technology   
90                                           Insurance   
93       EdTech, Education, Information Services, SaaS   
95   B2B, Biometrics, Cyber Security, Fraud Detecti...   
101           Food Delivery, Food Processing, Internet   
102            E-Commerce, Food and Beverage, Internet   
104            EdTech, Education, Knowledge Management   
106                                 Financial Services   
108  Creative Agency, Crowdfunding, EdTech, Health ...   
114                 Financial Services, SaaS, Security   
115   Banking, Finance, Financial Services, Non Profit   
117                            B2B, E-Comm

In [ ]:
data18['Amount'] = data18['Amount'].str.replace(',', '')
data18

Company Name                                           Industry  \
0      TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1      Happy Cow Dairy                               Agriculture, Farming   
2           MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3          PayMe India                        Financial Services, FinTech   
4             Eunimart                 E-Commerce Platforms, Retail, SaaS   
..                 ...                                                ...   
521              Udaan   B2B, Business Development, Internet, Marketplace   
522  Happyeasygo Group                                    Tourism, Travel   
523             Mombay         Food and Beverage, Food Delivery, Internet   
524         Droni Tech                             Information Technology   
525            Netmeds         Biotechnology, Health Care, Pharmaceutical   

    Round/Series     Amount                          Location  \
0           Seed     250000       Bangalore, Karnataka, India   
1           Seed  ₹40000000        Mumbai, Maharashtra, India   
2       Series A  ₹65000000           Gurgaon, Haryana, India   
3          Angel    2000000       Noida, Uttar Pradesh, India   
4           Seed          —  Hyderabad, Andhra Pradesh, India   
..           ...        ...                               ...   
521     Series C  225000000       Bangalore, Karnataka, India   
522     Series A          —           Haryana, Haryana, India   
523         Seed       7500        Mumbai, Maharashtra, India   
524         Seed  ₹35000000        Mumbai, Maharashtra, India   
525     Series C   35000000        Chennai, Tamil Nadu, India   

                                         About Company  Year  
0    TheCollegeFever is a hub for fun, fiesta and f...  2018  
1    A startup which aggregates milk from dairy far...  2018  
2            Leading Online Loans Marketplace in India  2018  
3    PayMe India is an innovative FinTech organizat...  2018  
4    Eunimart is a one stop solution for merchants ...  2018  
..                                                 ...   ...  
521  Udaan is a B2B trade platform, designed specif...  2018  
522            HappyEasyGo is an online travel domain.  2018  
523  Mombay is a unique opportunity for housewives ...  2018  
524  Droni Tech manufacture UAVs and develop softwa...  2018  
525       Welcome to India's most convenient pharmacy!  2018  

[526 rows x 7 columns]

GitHub Repository and Cloning to VSO Python Notebook:

In [ ]:
data18['Amount'] = data18['Amount'].str.replace(',', '')
data18
def convert_to_dollars(Amount):
    if Amount.startswith('₹'):
        return float(Amount[1:]) * 0.0117
    else:
        return Amount
 
# Apply the conversion function to the 'amount' column
data18['Amount'] = data18['Amount'].apply(convert_to_dollars)
data18





Company Name                                           Industry  \
0      TheCollegeFever  Brand Marketing, Event Promotion, Marketing, S...   
1      Happy Cow Dairy                               Agriculture, Farming   
2           MyLoanCare   Credit, Financial Services, Lending, Marketplace   
3          PayMe India                        Financial Services, FinTech   
4             Eunimart                 E-Commerce Platforms, Retail, SaaS   
..                 ...                                                ...   
521              Udaan   B2B, Business Development, Internet, Marketplace   
522  Happyeasygo Group                                    Tourism, Travel   
523             Mombay         Food and Beverage, Food Delivery, Internet   
524         Droni Tech                             Information Technology   
525            Netmeds         Biotechnology, Health Care, Pharmaceutical   

    Round/Series     Amount                          Location  \
0           Seed     250000       Bangalore, Karnataka, India   
1           Seed   468000.0        Mumbai, Maharashtra, India   
2       Series A   760500.0           Gurgaon, Haryana, India   
3          Angel    2000000       Noida, Uttar Pradesh, India   
4           Seed          —  Hyderabad, Andhra Pradesh, India   
..           ...        ...                               ...   
521     Series C  225000000       Bangalore, Karnataka, India   
522     Series A          —           Haryana, Haryana, India   
523         Seed       7500        Mumbai, Maharashtra, India   
524         Seed   409500.0        Mumbai, Maharashtra, India   
525     Series C   35000000        Chennai, Tamil Nadu, India   

                                         About Company  Year  
0    TheCollegeFever is a hub for fun, fiesta and f...  2018  
1    A startup which aggregates milk from dairy far...  2018  
2            Leading Online Loans Marketplace in India  2018  
3    PayMe India is an innovative FinTech organizat...  2018  
4    Eunimart is a one stop solution for merchants ...  2018  
..                                                 ...   ...  
521  Udaan is a B2B trade platform, designed specif...  2018  
522            HappyEasyGo is an online travel domain.  2018  
523  Mombay is a unique opportunity for housewives ...  2018  
524  Droni Tech manufacture UAVs and develop softwa...  2018  
525       Welcome to India's most convenient pharmacy!  2018  

[526 rows x 7 columns]

In [ ]:
data18.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 526 entries, 0 to 525
Data columns (total 7 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Company Name   526 non-null    object
 1   Industry       526 non-null    object
 2   Round/Series   526 non-null    object
 3   Amount         526 non-null    object
 4   Location       526 non-null    object
 5   About Company  526 non-null    object
 6   Year           526 non-null    int64 
dtypes: int64(1), object(6)
memory usage: 28.9+ KB


In [ ]:
# Rename round_series to stage and location to headquarter
data18.rename(columns={
    'Company Name': 'company_brand', 
    'Industry': 'sector', 
    'Round/Series': 'stage', 
    'About Company': 'what_it_does', 
    'Location': 'headquarter'
    },
    inplace=True
)

data18.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 526 entries, 0 to 525
Data columns (total 7 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   company_brand  526 non-null    object
 1   sector         526 non-null    object
 2   stage          526 non-null    object
 3   Amount         526 non-null    object
 4   headquarter    526 non-null    object
 5   what_it_does   526 non-null    object
 6   Year           526 non-null    int64 
dtypes: int64(1), object(6)
memory usage: 28.9+ KB


In [ ]:
# git clone https://github.com/SamuelAsong/indian-startup-funding-analysis.git
# cd indian-startup-funding-analysis
# pip install -r requirements.txt


CRISP-DM Process:
Business Understanding:

Define project objectives and requirements
Understand the start-up ecosystem and the importance of funding data
Data Understanding:

Gather and explore datasets for 2018, 2019, 2020, and 2021
Identify key features and initial insights
Data Preparation:

Clean and preprocess data
Handle missing values, duplicates, and inconsistent data
Merge datasets into a single comprehensive dataset
Data Analysis:

Perform exploratory data analysis (EDA)
Identify trends, patterns, and outliers
Visualize funding trends over the years
Modeling (if applicable):

Develop machine learning models to predict funding success (optional)
Evaluate model performance
Evaluation:

Assess the analysis results and model performance
Validate findings against business objectives
Deployment:

Present findings and recommendations
Prepare a final report and presentation
Conclusion and Findings:
Summarize key insights from the data analysis
Highlight significant trends and patterns in the Indian start-up funding landscape
Provide actionable recommendations based on data-driven insights
Discuss potential limitations and future work
This structured approach ensures a comprehensive analysis and effective communication of results, helping to make strategic, data-driven decisions in the Indian start-up ecosystem.








The CRISP-DM reference model 
1 Business understanding 

1.1 Determine business objectives 

1.2 Assess situation 

1.3 Determine data mining goals 

1.4 Produce project plan 

2 Data understanding 

2.1 Collect initial data 

2.2 Describe data 

2.3 Explore data 

2.4 Verify data quality 

3 Data preparation 

3.1 Select data 

3.2 Clean data 

3.3 Construct data 

3.4 Integrate data 

3.5 Format data 

4 Modeling 

4.1 Select modeling technique 

4.2 Generate test design 

4.3 Build model 

4.4 Assess model 

5 Evaluation 

5.1 Evaluate results 

5.2 Review process 

5.3 Determine next steps 

6 Deployment 

6.1 Plan deployment 

6.2 Plan monitoring and maintenance report 

6.4 Review project 